src/libcommonserver/io/iohelper.cpp

```diff
@@ -284,8 +285,12 @@ bool IoHelper::getItemType(const SyncPath &path, ItemType &itemType) noexcept {
     const bool isSymlink = _isSymlink(path, ec);
 
     itemType.ioError = stdError2ioError(ec);
+#ifndef WIN32
     const bool fsSupportsSymlinks =
             itemType.ioError != IoError::InvalidArgument; // If true, we assume that the file system in use does support symlinks.
+#else
+    const bool fsSupportsSymlinks = Utility::isNtfs(path.parent_path());
```

Comment:
What is the plan in terms of unit tests? (Another PR?) The fact that we are using static methods in helper classes make them difficult to test through mocking.